shub/config.py

```diff
@@ -81,11 +82,12 @@ def load(self, stream):
                 yaml_option_conf = yaml_cfg.get(option, {})
                 option_conf.update(yaml_option_conf)
                 if option == 'images' and yaml_option_conf:
-                    click.echo(
+                    warnings.warn(
```

Comment:
Addressing https://github.com/scrapinghub/shub/pull/253#issuecomment-293571505 and https://github.com/scrapinghub/shub/pull/253#issuecomment-293572056, I replaced click print to stderr with a usual warning.

But the real reason behind this issue is that each `load_shub_config()` command opens and parses a config file from scratch (sometimes even a few configs - global and local). Generally the workflow is fine per command, but if we call `shub deploy` with custom images - it's converted to a chain of commands, each one parsing same config again and again. A way better approach would be to read it only once, and pass parsed config to other commands as an optional parameter and load config file only if the param is not passed, I'll prepare the changes it if you agree.